### PR TITLE
Add mentor notes for csharp track isogram

### PR DESCRIPTION
**Pull request by fpetru**
Please help to review this mentoring note for isogram exercise. Thank you in advance.

### CODE REVIEW

File: tracks/csharp/exercises/isogram/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Common suggestions
+
+- In the non LINQ version, try to use a single loop
+
+- Using LINQ next suggestions could be made:
+  * each element of the string should be a letter (in this way we avoid checking for space or '-')
+  * without making any iteration, _distinct_ method helps to enforce the letter uniqueness, and find the isogram   
+    
+### Talking points
```
Comment (ErikSchierboom) [inline tracks/csharp/exercises/isogram/mentoring.md]: As there are no talking point, I would suggest to remove this header.

File: tracks/csharp/exercises/isogram/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        return trimmedInput.Distinct().Count() == trimmedInput.Count();
+    }
+}
+```
+
+### Common suggestions
+
+- In the non LINQ version, try to use a single loop
+
+- Using LINQ next suggestions could be made:
```
Comment (ErikSchierboom) [inline tracks/csharp/exercises/isogram/mentoring.md]:
```suggestion
- When using LINQ, the following suggestions can be made:
```

File: tracks/csharp/exercises/isogram/mentoring.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  * each element of the string should be a letter (in this way we avoid checking for space or '-')
```
Comment (ErikSchierboom) [inline tracks/csharp/exercises/isogram/mentoring.md]: Good suggestion, maybe a slight rewording:

```suggestion
  * To make things easier, try only processing the string's letters (in this way we avoid checking for spaces or '-').
```

File: tracks/csharp/exercises/isogram/mentoring.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  * without making any iteration, _distinct_ method helps to enforce the letter uniqueness, and find the isogram
```
Comment (ErikSchierboom) [inline tracks/csharp/exercises/isogram/mentoring.md]: Small suggestion:

```suggestion
  * Suggest using the `Distinct()` method to help calculate the uniqueness of the letters.
```

File: tracks/csharp/exercises/isogram/mentoring.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+```csharp
+using System;
+using System.Linq;
+
+public static class Isogram
+{
+    public static bool IsIsogram(string word)
+    {
+        // Ignore spaces, hyphens, regard uppercase as lower.
+        var trimmedInput = word.ToLower().ToCharArray().Where(Char.IsLetter);
```
[the remark this answers is not in the document]
Comment (fpetru) [reply tracks/csharp/exercises/isogram/mentoring.md]: true, done

File: tracks/csharp/exercises/isogram/mentoring.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        return trimmedInput.Distinct().Count() == trimmedInput.Count();
```
[the remark this answers is not in the document]
Comment (fpetru) [reply tracks/csharp/exercises/isogram/mentoring.md]: yes, this was the reason why I switched the order earlier to make it more readable (comparing Count() with Count()). But it is true, it's better. Done

File: tracks/csharp/exercises/isogram/mentoring.md (hunk 3 above)
Comment (ErikSchierboom) [reply tracks/csharp/exercises/isogram/mentoring.md]: This was marked as resolved, but I did not see any change. Was that deliberate?

File: tracks/csharp/exercises/isogram/mentoring.md (hunk 1 above)
Comment (fpetru) [reply tracks/csharp/exercises/isogram/mentoring.md]: sure, done

**Comment by fpetru** [comment]
Thank a lot for you review @ErikSchierboom !

File: tracks/csharp/exercises/isogram/mentoring.md (hunk 3 above)
Comment (fpetru) [reply tracks/csharp/exercises/isogram/mentoring.md]: sorry, I have simply missed it. I've just committed it now.

Comment (ErikSchierboom) [reply tracks/csharp/exercises/isogram/mentoring.md]: No problem! It's easy to miss things when there are a bunch of comments.

**Comment by ErikSchierboom** [comment]
Merged! Once again thanks a lot @fpetru!

**Pull request merged**